controllers/mock.go

```diff
@@ -329,6 +329,9 @@ func MockCoduno(w http.ResponseWriter, req *http.Request) {
 	}
 
 	model.Test{
-		Tester: int64(test.Simple),
+		Tester: int64(test.Junit),
+		Params: map[string]string{
+			"test": "junit",
```

Comment:
Why is this needed? It seems redundant to specify the tester and copy it's name here.